app/backend/approaches/retrievethenread.py

```diff
@@ -13,39 +14,29 @@ class RetrieveThenReadApproach(Approach):
     (answer) with that prompt.
     """
 
-    template = \
+    system_chat_template = \
 "You are an intelligent assistant helping Contoso Inc employees with their healthcare plan questions and employee handbook questions. " + \
 "Use 'you' to refer to the individual asking the questions even if they ask with 'I'. " + \
 "Answer the following question using only the data provided in the sources below. " + \
 "For tabular information return it as an html table. Do not return markdown format. "  + \
 "Each source has a name followed by colon and the actual information, always include the source name for each fact you use in the response. " + \
-"If you cannot answer using the sources below, say you don't know. " + \
-"""
+"If you cannot answer using the sources below, say you don't know. Use below example to answer"
 
-###
-Question: 'What is the deductible for the employee plan for a visit to Overlake in Bellevue?'
+    #shots/sample conversation
```

Comment:
nit: please expand this comment so it provides a tiny bit of context/explanation, or remove.